src/three/TilesRenderer.js

```diff
@@ -939,10 +950,12 @@ export class TilesRenderer extends TilesRendererBase {
 				} else {
 
 					tempVector.applyMatrix4( transformInverse );
+					tempSphere.copy( boundingSphere );
+					tempSphere.applyMatrix4( transformInverse );
 					// Sphere#distanceToPoint is negative inside the sphere, whereas Box3#distanceToPoint is
 					// zero inside the box. Clipping the distance to a minimum of zero ensures that both
 					// types of bounding volume behave the same way.
-					distance = Math.max( boundingSphere.distanceToPoint( tempVector ), 0 );
+					distance = Math.max( tempSphere.distanceToPoint( tempVector ), 0 );
```

Comment:
Since we save a transformed bounding sphere this was causing the calculate error to always be wrong for bounding spheres when a transform matrix was present. This applies the inverseTransform to the bounding sphere before calculating the distance.